annotation-server/src/common/google-sheets.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { Test, TestingModule } from '@nestjs/testing';
+
+import { fetchSpreadsheetCells } from './google-sheets';
+
+describe('Helper to fetch spreadsheet', () => {
+    let configService: ConfigService;
+
+    beforeEach(async () => {
+        const modelFixture: TestingModule = await Test.createTestingModule({
+            imports: [
+                ConfigModule.forRoot({ envFilePath: ['test/.env', '.env'] }),
+            ],
+        }).compile();
+
+        configService = modelFixture.get<ConfigService>(ConfigService);
+    });
+
+    describe('fetchSpreadsheetCells', () => {
+        it('should not return an error with valid sheet id', async () => {
```

Comment:
I think the name is a little complicated; maybe consider something like `should get response with data from Google Sheets API`